sympy/core/add.py

```diff
@@ -1127,6 +1124,7 @@ def __neg__(self):
             return super().__neg__()
         return Add(*[-i for i in self.args])
 
+add = AssocOpDispatcher('add')
```

Comment:
Can you add a small `doc` argument?
```suggestion
add = AssocOpDispatcher('add', doc="""
...
""")
```
```

Reply:
``AssocOpDispatcher.__doc__ is implemented instead.``

Reply:
Do you plan on integrating this doc argument with SymPy's documentation?

Reply:
I would like to treat this as an experimental feature for now so if it's documented it should be clearly marked. It needs to be clear in the code and especially in the docs that it is experimental and there are no backwards compatibility guarantees - we might change this in a future release.

Reply:
@Upabjojr Not really. Every detailed information is already delineated in `AssocOpDispatcher`'s docstring.

Reply:
Will `add.__doc__` end up in the sphinx html docs somewhere?

Reply:
I don't know sphinx well. Anyone, please?

Reply:
I don't think so. I don't think anything in this module goes into the sphinx docs.

Reply:
There is `doc/src/modules/core.rst` but that doesn't pull anything in from here